LICENSE.md

```diff
@@ -0,0 +1,12 @@
+About MuleSoft Certified Connectors
+
+Mulesoft Certified Connectors are developed by MuleSoft's partners and developers community. These connectors have been reviewed and certified by Mulesoft. To purchase the MuleChain AI Connector or to receive assistance or support for it, contact MuleChain Project team. MuleSoft disclaims any support obligation for MuleSoft Connectors.
```

Comment:
The below line is still not valid. So we can add this later after completion of certification.
"These connectors have been reviewed and certified by Mulesoft"